app/models/ext_management_system.rb

```diff
@@ -578,6 +578,11 @@ def stop_event_monitor
   end
 
   def stop_event_monitor_queue
+    unless id
```

Comment:
Prefer `unless new_record?`, as that matches the code we have [here](https://github.com/ManageIQ/manageiq/pull/8785/files#diff-bc914c1dea04b4ba49055f79a8ab8b5aR597) and [here](https://github.com/ManageIQ/manageiq/pull/8785/files#diff-bc914c1dea04b4ba49055f79a8ab8b5aR604) as well.  In fact, why are those existing checks not handling this case?